meditool/examinations/emg/features.py

```diff
@@ -0,0 +1,31 @@
+import meditool.data_types.signal.features as sfeatures
+from typing import Union
+import numpy as np
+
+def arc_length(signal: np.ndarray, sample_rate: Union[int, float]) -> float:
+    """Method for calcualting the arc length of an EMG signal using Euclidian distance.
+	:param signal: The signal represented as 1 dimensional numpy array
+	:type signal: np.ndarray
+	:param sample_rate: Sample rate of a signal used to calculate Euclidean distance between following samples
+	:type sample_rate: Union[int, float]
+	:return: Float. Calculated arc length  
+	:rtype: float
+    """
+    if type(signal) == np.ndarray) and type(sample_rate) == (int or float):
```

Comment:
Brakuje jednego nawiasu z lewej strony :) (albo niepotrzebny nawias zamykający przed `and`)

A warunek po `and` nie zadziała, bo `(int or float)` zwraca `int`, lepiej dać: `type(sample_rate) in (int, float)`.

Reply:
Już poprawione